projects/packages/forms/src/util/compute-slider-position.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Compute the CSS left value for the slider value indicator so it stays centered
+ * above the thumb. Matches the formula used on the frontend interactivity store.
+ *
+ * @param {number|string} minParam   - Minimum slider value
+ * @param {number|string} maxParam   - Maximum slider value
+ * @param {number|string} valueParam - Current slider value
+ * @return {string} CSS calc expression for the left position
+ */
+export function computeSliderPosition( minParam, maxParam, valueParam ) {
+	const min = Number( minParam );
+	const max = Number( maxParam );
+	let value = Number( valueParam );
+
+	// Fallbacks and clamping
+	value = Number.isNaN( value ) ? min : value;
+	value = Math.max( min, Math.min( value, max ) );
+
+	const percent = ( ( value - min ) * 100 ) / ( max - min );
+	// Magic numbers: 16px base offset, 0.32px per percent
+	return `calc(${ percent }% + (${ 16 - percent * 0.32 }px))`;
+}
```

Comment:
If there's a better place for shared JS utility methods to live, or if we would rather create a catchall util.js file and put methods in that, let me know.

Reply:
Since the util is specific to the slider field (with magic numbers and all), might make sense to keep it in the slider field folder and share between front/back from there?

Reply:
Yes, we could do that. In terms of code organization, the code is used in in both blocks and module folders here:
forms/src/blocks/input-range/edit.js
forms/src/modules/field-slider/view.js

One thing I've ben meaning to ask, is why we put view.js in a separate modules folder? I think traditionally in block development, a blocks view.js (it's frontend JS) just lives in the block folder along with other block related code. 

Any opposition to moving the view.js to the block folder (may have to update some config somewhere too)? Then frontend JS, editor JS, and any utils would all be consolidated.

Reply:
There's a separate discussion on slack now about view.js loading from block folder. For this PR, to keep things moving, I've just moved the shared method to a util.js folder in the input-range block folder. So it lives close to the block code, and then we just import it to view.js from there. Hopefully we'll eventually move view.js to the block folder too. 

Since I already had approval on this PR and just move the method definition, once tests pass, I'll go ahead and merge.